### ISSUE TITLE
Keyboard malfunctioning in specific use case. It seems to be opening and closing.

### ISSUE THREAD

**Issue by diegovfeder** [react-native-webview/react-native-webview#1824]
**Bug description:**

I noticed a strange behaviour for a very specific case while using this library. Keyboard flashes when clicking on a select box that is inside a modal. I'm uploading a video that demonstrates this odd behaviour. 
The problem seems to be caused by the select box or because there is a modal opening another modal and the user action opens a keyboard sequentially. 

**To Reproduce:**

The important thing here is the uri: https://www.att.com/buy/bundles/hsiaplans. When you open the listed link, a Modal should be prompted waiting for the user to input an address. If this address contains multiple units, for example: 3 Olvera St, Los Angeles, CA 90012, the webpage will prompt another modal and here comes the problem. When I try to click the select box to pick an unit the keyboard will flash (open and close very fast) making it impossible for the user to finish this process. 

Code I'm using:
```
        <WebView
          style={styles.webView}
          source={{uri: 'https://www.att.com/buy/bundles/hsiaplans'}}
          incognito={true}
          startInLoadingState={true}
          renderLoading={() => <Loading size={96} />}
        />
```

**Expected behavior:**

The expected behaviour would be to either have the unit select box to open without even triggering the keyboard or to toggle(open) the keyboard and wait for user action without flashing (opening and closing keyboard very fast).

**Screenshots/Videos:**

This is what should happen when the select box is clicked:

![image](https://user-images.githubusercontent.com/63731464/104503435-d7a59680-55bf-11eb-9f1a-14d2385dfcb9.png)

This is a link for a video demonstrating the issue:
- mov
https://user-images.githubusercontent.com/63731464/104505721-1e48c000-55c3-11eb-957d-349e85486413.mov
- mp4
https://user-images.githubusercontent.com/63731464/104508788-83061980-55c7-11eb-8c0e-121e102ad190.mp4

**Environment:**
This only happens in iOS (mobile and tablet). Android works perfectly!
 - OS: iOS
 - OS version:  11
 - react-native version: 0.63.4
 - react-native-webview version: "^10.10.2"
 
 Please let me know if you need more information on the issue or anything.
 Thank you 🙏

**Comment by diegovfeder** [comment]
No solution yet

**Comment by jyotigupta007** [comment]
Hello, 
I am also facing same issue. Have you found any solutions

**Comment by diegovfeder** [comment]
> Hello,
> I am also facing same issue. Have you found any solutions

Hey. Sorry I didn't read your comment before.
And actually no I did not find any solution... I just mentioned the issue to my client and we moved forward with development.
Two things I also mentioned to them was:
1. Open the user default browser
2. The possibility to re-do the needed web logic/code-flow inside React Native using same endpoint or w/e... 
but I didn't find any solution using this lib

What about you? Did you find any solution for this?
